docs/versioned_docs/version-2.x/components.mdx

```diff
@@ -698,7 +719,11 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
 
 * **Requires**
 
-  [HFTransformersNLP](./components.mdx#hftransformersnlp) and [LanguageModelTokenizer](./components.mdx#languagemodeltokenizer)
+  `tokens`.
+
+  :::caution Changed in 2.1
```

Comment:
ok I think I get the strategy now but wanted to confirm: the main docs will show the current behavior, and the caution flags will say around it that it changed in 2.x.x minor

Reply:
yes exactly. So if you're a user of 2.4 and 2.8 is the latest version out on the 2.x major, you'll see the doc of 2.8, with important information about things that changes in versions of the 2.x major. I think it's also great because it prompts users to think about upgrading when they're reading the docs (that's my train of thoughts when I read Python docs and I see a useful feature that I cannot use yet, for instance)

Reply:
awesome